runsc/fsgofer/filter/config.go

```diff
@@ -83,6 +83,10 @@ var allowedSyscalls = seccomp.SyscallRules{
 			seccomp.AllowAny{},
 			seccomp.AllowValue(syscall.F_GETFD),
 		},
+		{
```

Comment:
Why is F_DUPFD_CLOEXEC needed when fsgofer UDS is disabled?

Reply:
This actually isn't needed anymore anyways! This was leftover from the original implementation on `net` I believe! Good catch :)